src/components/Wishlist/Wishlist.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { IconButton, Accordion } from '@material-ui/core/';
+import Wishlist from './Wishlist';
+import { handleDeleteWishlist } from '../../client/client';
+
+describe('Wishlist', () => {
+  let component;
+  const wishlist = {
+    id: 1,
+    name: "Kendra's Wishlist",
+    author: 'Kendra',
+    items: [
+      { name: 'No More EPRs' },
+      { name: 'to understand TDD' },
+      { name: 'Fun' },
+    ],
+  };
+
+  const mockHandleDeleteWishlist = jest.fn();
+
+  beforeEach(() => {
+    mockHandleDeleteWishlist.mockImplementation(handleDeleteWishlist);
+    component = mount(
+      <Wishlist
+        wishlist={wishlist}
+        handleDeleteWishlist={mockHandleDeleteWishlist}
+      />
+    );
+  });
+
+  it('should load wishlist', () => {
+    expect(component.text()).toContain(wishlist.name);
+    expect(component.text()).toContain(wishlist.author);
+    expect(component.text()).toContain(wishlist.items[0].name);
+    expect(component.text()).toContain(wishlist.items[1].name);
+    expect(component.text()).toContain(wishlist.items[2].name);
```

Comment:
Good attention to detail! It's good to be thorough and check all of these - after all, trusting that everything worked by checking ONE part of the Wishlist's ability to render the wishlist object it was passed as props might not mean it was doing EVERYTHING it was supposed to!